doc/howtos/backend/exercise-translations

```diff
@@ -48,7 +48,7 @@ diff --git a/openacademy/models.py b/openacademy/models.py
                  },
              }
  
-@@ -132,4 +132,4 @@ class Session(models.Model):
+@@ -114,4 +114,4 @@ class Session(models.Model):
```

Comment:
I will trust you on this one. @madprog is the expert in patchqueue if you need helps

Reply:
I'm fixing those errors in a "stupid" way without any patchqueue program, but it works nonetheless :grinning:  (I already spoke about the doc patches with Paul, I know he's the new Odoo patchqueue expert ^^).